lib/showoff.rb

```diff
@@ -303,25 +304,25 @@ def process_markdown(name, section, content, opts={:static=>false, :pdf=>false,
       # todo: unit test
       lines = content.split("\n")
       @logger.debug "#{name}: #{lines.length} lines"
-      slides = []
-      slides << (slide = Slide.new)
+      slideset = []
```

Comment:
Yeah that can be pulled out and it doesn't necessary need to happen. I changed it because I was running into an error about "slides" at some point and I realized I couldn't tell which thing it was referring to. I just left it in since I don't think this PR should really get merged.